test/functional/test_runner.py

```diff
@@ -54,104 +54,104 @@
 BASE_SCRIPTS= [
     # Scripts that are run by the travis build process.
     # Longest test should go first, to favor running tests in parallel
-    'wallet-hd.py',
-    'walletbackup.py',
+    'wallet_hd.py',
+    'wallet_backup.py',
     # vv Tests less than 5m vv
-    'p2p-fullblocktest.py',
-    'fundrawtransaction.py',
-    'p2p-compactblocks.py',
-    'segwit.py',
+    'p2p_block.py',
```

Comment:
p2p_block tests block acceptance and reorg behaviour rather than the p2p protocol so should probably be in the feature_ namespace